static/app/views/projectInstall/platform.tsx

```diff
@@ -136,6 +141,11 @@ export function ProjectInstallPlatform({location, params, route, router}: Props)
     ? projects.find(proj => proj.slug === params.projectId)
     : undefined;
 
+  const products = useMemo(
+    () => (location.query.product ?? []) as PRODUCT[],
+    [location.query.product]
+  );
```

Comment:
Do you really need useMemo for this? What are you computing?

Reply:
This can just be 

```tsx
const products: PRODUCT[] = location.query.product ?? []
```

Reply:
It doesn't look like but I was getting a warning

Reply:
What was the warning?

Reply:
it was like that before and because of the warning (it is a dependency of a useEffect) I applied the suggestion

Reply:
tried that before but 

<img width="1044" alt="image" src="https://github.com/getsentry/sentry/assets/29228205/f2208b0e-ff34-4875-a7ae-666d9a7c0b4f">